specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -185,7 +187,7 @@ int main()
     }
 
     // Unregisters the inproc COM Activator before exiting
-    PushNotificationManager::UnregisterActivator(token, PushNotificationRegistrationKind::ComActivator);
+    PushNotificationManager::UnregisterActivator(token, pushOptions);
```

Comment:
If you pass the flags as is and it is ProtocolActivator, we prevent the app to get notifications, which we don't want.

Reply:
There needs to be a check if we returned Protocol/ComActivator from GetPushActivatorFlags. And unregister accordingly.